------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-10.csv'
[Errno 2] No such file or directory: '2013-10.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 5] Input/output error[Errno 2] No such file or directory: '2013-09.csv'
[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 2] No such file or directory: '2013-09.csv'
[Errno 5] Input/output error------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 5] Input/output error[Errno 2] No such file or directory: '2013-09.csv'
[Errno 5] Input/output error------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ashley/thesis/monitoring.py", line 228, in <module>
    main()
  File "/home/ashley/thesis/monitoring.py", line 61, in main
    dbHelper.get_file('config.ini',path+'/config.ini')
  File "/home/ashley/Desktop/thesis/RaspPi/Dropbox.py", line 80, in get_file
    f, metadata = self.api_client.get_file_and_metadata(self.current_path + "/" + from_path)
  File "build/bdist.linux-armv6l/egg/dropbox/client.py", line 206, in get_file_and_metadata
  File "build/bdist.linux-armv6l/egg/dropbox/client.py", line 183, in get_file
  File "build/bdist.linux-armv6l/egg/dropbox/rest.py", line 261, in request
  File "build/bdist.linux-armv6l/egg/dropbox/rest.py", line 153, in request
  File "/usr/lib/python2.7/httplib.py", line 962, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python2.7/httplib.py", line 996, in _send_request
    self.endheaders(body)
  File "/usr/lib/python2.7/httplib.py", line 958, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 818, in _send_output
    self.send(msg)
  File "/usr/lib/python2.7/httplib.py", line 780, in send
    self.connect()
  File "build/bdist.linux-armv6l/egg/dropbox/rest.py", line 42, in connect
  File "/usr/lib/python2.7/ssl.py", line 381, in wrap_socket
    ciphers=ciphers)
  File "/usr/lib/python2.7/ssl.py", line 143, in __init__
    self.do_handshake()
  File "/usr/lib/python2.7/ssl.py", line 305, in do_handshake
    self._sslobj.do_handshake()
KeyboardInterrupt
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
No echo received
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
No echo received
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
Cannot sychronise time
Traceback (most recent call last):
  File "/home/ashley/thesis/monitoring.py", line 228, in <module>
    main()
  File "/home/ashley/thesis/monitoring.py", line 61, in main
    dbHelper.get_file('config.ini',path+'/config.ini')
  File "/home/ashley/Desktop/thesis/RaspPi/Dropbox.py", line 80, in get_file
    f, metadata = self.api_client.get_file_and_metadata(self.current_path + "/" + from_path)
  File "build/bdist.linux-armv6l/egg/dropbox/client.py", line 206, in get_file_and_metadata
  File "build/bdist.linux-armv6l/egg/dropbox/client.py", line 183, in get_file
  File "build/bdist.linux-armv6l/egg/dropbox/rest.py", line 261, in request
  File "build/bdist.linux-armv6l/egg/dropbox/rest.py", line 188, in request
dropbox.rest.RESTSocketError: Error connecting to "api-content.dropbox.com": [Errno -2] Name or service not known
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
[Errno 2] No such file or directory: '2013-09.csv'
------------------------------------------------------------------------
------------------------------------------------------------------------
